docs/trex_events.md

```diff
@@ -44,5 +44,4 @@ In most cases, you can create an event listener by chaining the methods to the s
 // ...
 ```  
 
-For more information, check out the sample extension, Filtering.
-
+For more information, check out the sample extension, [Filtering](https://github.com/tableau/extensions-api/tree/main/Samples/Filtering).
```

Comment:
Can you use a relative link here? Relative link will figure out current branch and stuff.

Reply:
I hear you. It would rather not use an absolute path, but I think we might have to in this case. The Help pages are served up at [https://tableau.github.io/extensions-api/](https://tableau.github.io/extensions-api/).  If you append `Samples/Filtering` to the path ([https://tableau.github.io/extensions-api/Samples/Filtering/](https://tableau.github.io/extensions-api/Samples/Filtering/)), you get an error.
On the other hand, it sort of works when I test it locally. In that case, it's just looking at the file system.  Bottom line is I don't know what will happen. :)  until we actually merge with the main branch. That's the only way to know for sure.

Reply:
alright. leave it. Closing the other stuff too.